api_test.py

```diff
@@ -0,0 +1,21 @@
+import requests
+
+BEARER = None
```

Comment:
Be careful with secrets, never push them to origin!

Reply:
ok